src/gtc/passes/oir_pipeline.py

```diff
@@ -0,0 +1,72 @@
+from typing import Callable, Dict, Optional, Sequence, Tuple
```

Comment:
Header here.

Reply:
I'll stop commenting that, but there are a number of other files that still require it.